### ISSUE TITLE
Member detail page: Ajax error when viewing guest

### ISSUE THREAD

**Issue by kduncklee** [BAMRU-Tech/bamru_net#311]
When viewing a member that has no certs (like a guest), the member_certs api call returns a 404 because there are no matching certs.

Observed: This causes the page to display a warning dialog.

Expected: The page should gracefully handle the 404 and show an empty table.

**Comment by ctaylor451** [comment]
This seems like a backend bug.  If certs are requested the expected result is an empty cert list

**Comment by kduncklee** [comment]
Yes, you're right. It turns out that the backend was filtering on current members.
